src/__test__/components/data-exploration/generic-gene-table/PathwayAnalysisModal.test.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import PathwayAnalysisModal from 'components/data-exploration/differential-expression-tool/PathwayAnalysisModal';
+import {
+  render, screen, fireEvent, waitFor,
+} from '@testing-library/react';
+
+describe('Pathway analysis modal ', () => {
+  const onCancel = jest.fn();
+  const renderPathwayAnalysisModal = () => {
+    render(<PathwayAnalysisModal onCancel={onCancel} />);
+  };
+
+  it('Renders properly', () => {
+    renderPathwayAnalysisModal();
+    expect(screen.getByText('You have not performed any filtering on the genes!')).toBeInTheDocument();
+    expect(screen.getByLabelText('pantherdb')).toBeInTheDocument();
+    expect(screen.getByLabelText('pantherdb').checked).toEqual(true);
+    expect(screen.getByLabelText('enrichr')).toBeInTheDocument();
```

Comment:
Maybe put the name of the services into a constant and loop over them to check? That would make it easier to extend if we're adding more.